aggregator_core/src/datastore.rs

```diff
@@ -6847,23 +6849,30 @@ mod tests {
                 // Write a few aggregation jobs we expect to be able to retrieve with
                 // acquire_incomplete_aggregation_jobs().
                 tx.put_task(&task).await?;
-                for aggregation_job_id in aggregation_job_ids {
-                    tx.put_aggregation_job(&AggregationJob::<
-                        PRIO3_VERIFY_KEY_LENGTH,
-                        TimeInterval,
-                        Prio3Count,
-                    >::new(
-                        *task.id(),
-                        aggregation_job_id,
-                        (),
-                        (),
-                        Interval::new(Time::from_seconds_since_epoch(0), Duration::from_seconds(1))
+                try_join_all(aggregation_job_ids.into_iter().map(|aggregation_job_id| {
```

Comment:
It occurs to me that there are likely a number of places where we iterate over a long list of reports or report aggregations or aggregation jobs or whatever like this, and could get performance wins either by spawning a bunch of tokio tasks or using [`rayon::iter`](https://docs.rs/rayon/latest/rayon/iter/index.html). No need to change that here: #1307